client/signup/steps/domains/index.jsx

```diff
@@ -193,6 +224,10 @@ class DomainsStep extends Component {
 		} );
 	};
 
+	isExperiment() {
+		return this.state.experiment?.variationName === 'treatment';
```

Comment:
Minor: As per the [ExPlat documentation](https://github.com/Automattic/wp-calypso/blob/trunk/packages/explat-client/README.md), the correct way to check for treatment is to verify that variationName is not null, instead of checking equality for `treatment`. 

```
variationName !== null: This means you should return the treatment experience. Currently variationName will always be treatment, but this may change.
```